test/service/test_app_logging.py

```diff
@@ -51,14 +51,20 @@ def test_request_logs(self):
                         'connection': 'keep-alive',
                         **headers,
                     }
+                    self_ = sq('self')
+                    none = sq('none')
                     response_headers = {
                         'Access-Control-Allow-Origin': '*',
                         'Access-Control-Allow-Headers': 'Authorization,'
                                                         'Content-Type,'
                                                         'X-Amz-Date,'
                                                         'X-Amz-Security-Token,'
                                                         'X-Api-Key',
-                        'Content-Security-Policy': f"default-src {sq('self')}",
+                        'Content-Security-Policy': f"default-src {self_};"
```

Comment:
With the inner quotes extracted, the double quotes in these paragraphs should be converted to single quotes in this commit.